### ISSUE TITLE
Bug in Heron USV simulations

### ISSUE THREAD

**Issue by MNikoliCC** [cyberbotics/webots#6716]
**Describe the Bug**
Start the `swarm.wbt` or `ocean.wbt `worlds and the simulation will not start (only black screen).
The following warnings appear:
```
ERROR: Error downloading EXTERNPROTO 'OilBarrel': Cannot download 'https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/obstacles/protos/OilBarrel.proto', error code: 203: Error transferring https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/obstacles/protos/OilBarrel.proto - server replied: 
ERROR: Error downloading EXTERNPROTO 'TexturedBackground': Cannot download 'https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/backgrounds/protos/TexturedBackground.proto', error code: 203: Error transferring https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/backgrounds/protos/TexturedBackground.proto - server replied: 
ERROR: Error downloading EXTERNPROTO 'TexturedBackgroundLight': Cannot download 'https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto', error code: 203: Error transferring https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto - server replied: 
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':14:1: error: Skipped unknown 'TexturedBackground' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':16:1: error: Skipped unknown 'TexturedBackgroundLight' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':40:1: error: Skipped unknown 'OilBarrel' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':56:1: error: Skipped unknown 'OilBarrel' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':73:1: error: Skipped unknown 'OilBarrel' node or PROTO.
ERROR: '/home/milos/webots/projects/robots/clearpath/heron/worlds/swarm.wbt':90:1: error: Skipped unknown 'OilBarrel' node or PROTO.
WARNING: Heron "heron":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron(3)":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron(1)":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron(2)":  Skipped node: Cannot insert Transform node in 'boundingObject' field of Robot node in bounding object.
WARNING: Heron "heron": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
WARNING: Heron "heron(3)": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
WARNING: Heron "heron(1)": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
WARNING: Heron "heron(2)": Mass is invalid because 'boundingObject' is not defined. Using default mass properties: mass = 1, inertia matrix = identity
WARNING: Heron "heron" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.
WARNING: Heron "heron(3)" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.
WARNING: Heron "heron(1)" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.
WARNING: Heron "heron(2)" > DEF HERON Shape > Mesh "": Too many triangles (23409) in mesh: unable to cast shadows, please reduce the number of triangles below 21845 or set Shape.castShadows to FALSE to disable this warning.
```

**Steps to Reproduce**
1. Start the one of the sample simulations: `swarm.wbt` or `ocean.wbt `

**Expected behavior**
Simulation runs smoothly, no warnings and errors.

**System**
 - Operating System: Ubuntu 22.04
 - Graphics Card: [RENOIR (renoir, LLVM 15.0.7, DRM 3.57, 6.8.0-49-generic)

**Comment by ShuffleWire** [comment]
You are running Webots 2025a, and "https://raw.githubusercontent.com/cyberbotics/webots/R2025a/projects/objects/obstacles/protos/OilBarrel.proto" don't exist (404 error)

I don't think resources have not been upgraded yet from 2024a to 2025a

if you can, try tweeking the url to use 2023b ressources instead

**Comment by lukicdarkoo** [comment]
@ShuffleWire Check the referenced PR. The EXTERN URLs should start with `webots://`, not hardcoded.

The fix in PR is fine, but Milos may fix a few more related issues inside the same PR.

**Comment by lukicdarkoo** [comment]
This is fixed in #6717
